test/unit/org/apache/cassandra/security/SSLFactoryTest.java

```diff
@@ -296,9 +311,38 @@ public void testCacheKeyInequalityForCustomSslContextFactory() {
         .withSslContextFactory(new ParameterizedClass(DummySslContextFactoryImpl.class.getName(), parameters2))
         .withProtocol("TLSv1.1");
 
-        SSLFactory.CacheKey cacheKey2 = new SSLFactory.CacheKey(encryptionOptions2, ISslContextFactory.SocketType.SERVER
+        SSLFactory.CacheKey cacheKey2 = new SSLFactory.CacheKey(encryptionOptions2, ISslContextFactory.SocketType.SERVER, "test"
         );
 
         Assert.assertNotEquals(cacheKey1, cacheKey2);
     }
+
+    public static class TestFileBasedSSLContextFactory extends FileBasedSslContextFactory {
```

Comment:
unused?

Reply:
accidentally included in back port, used by other tests in 5.0/trunk